### ISSUE TITLE
Scrolling is broken in the new chrome update

### ISSUE THREAD

**Issue by rtseuztz** [NeXTs/Clusterize.js#187]
When Scrolling through a clusterized list, whenever a new cluster (or something like that) is hit the list stops scrolling. To keep scrolling, I can move my mouse a little bit and then scrolling works again. Currently to scroll a whole list without stopping, I have to move my mouse slightly to the left and right while scrolling.

**Comment by MitchellSPeterson** [comment]
Chrome update: 102.0.5005.63
Seems to happen on cluster change.

**Comment by NeXTs** [comment]
I can not reproduce this on my mac.
Latest chrome version
<img width="325" alt="image" src="https://user-images.githubusercontent.com/1753208/171270140-d94a7d6f-826b-4620-8df4-2289eabb0f92.png">

Could you please check if this helps
https://jsfiddle.net/ktnbexa8/

**Comment by MitchellSPeterson** [comment]
Yeah, I've been messing around with something similar. It does help but isn't perfect, hopefully google puts out a fix soon.
![image](https://user-images.githubusercontent.com/69201534/171271657-e235765f-6c79-4ecc-8557-b65a5d68b65d.png)
